src/stories/Library/Lists/list-empty/ListEmpty.stories.tsx

```diff
@@ -0,0 +1,29 @@
+import React from "react";
+import { withDesign } from "storybook-addon-designs";
+import { ComponentMeta, ComponentStory } from "@storybook/react";
+import ListEmpty from "./ListEmpty";
+
+export default {
+  title: "Library / Lists / List - empty",
+  component: ListEmpty,
+  decorators: [withDesign],
+  argTypes: {
+    text: {
+      defaultValue: "Du har i øjeblikket 0 Reserveringer",
```

Comment:
Should default text be in english?

Reply:
They could I assume, I dont think its that important...? @spaceo perhaps has an opinion?

Reply:
I think the storybook default values should be in danish.
NB: Adam said that defaultValue will be deprecated in the future and we should use the object binded to the story component:
<img width="240" alt="image" src="https://user-images.githubusercontent.com/998889/178494734-fd3ebb3f-a805-4b5f-a6d3-7cee3e19cf78.png">